snafu/uperf_wrapper/uperf_wrapper.py

```diff
@@ -54,6 +54,13 @@ def __init__(self, parent_parser):
         self.args.num_pairs = os.getenv("num_pairs", "")
         self.args.multus_client = os.getenv("multus_client", "")
         self.args.networkpolicy = os.getenv("networkpolicy", "")
+        self.args.nodes_in_iter = os.getenv("node_count", "0")
+        self.args.pod_density = os.getenv("pod_count", "0")
+        self.args.colocate = os.getenv("colocate","False")
+        self.args.step_size = os.getenv("stepsize","")
+        self.args.density_range = os.getenv("density_range","0-0")
+        self.args.node_range = os.getenv("node_range","0-0")
```

Comment:
Aren't we exporting a list like [2,4]? if so, lets directly set node_range to a list type instead of string.

Reply:
I was planning on using this as metadata for a run, not planning on using the range feature of ES.

Reply:
I don't see how having this in the form you have current will help with anything in the future. even to do run tor un compairsons with touchstone or the likes in future, I'd prefer to keep node_range and desnity_range as lists.

Reply:
@smalleni Can you elaborate why? I ask because they are exported as bash environment variables which doesn't have a list type that I am aware of so the list aspect is gone. I don't know of a way that the list aspect would be useful in elastic search. After having gone though some iterations with benchmark-wrapper and benchmark-comparison the uuid will serve the purpose of identifying a run so maybe we don't need the variables ?

Reply:
As long as they are exported you can always slice and dice convert them to a list in python :-) 

I’m thinking of cases where you want to identify runs that had a node_range of a particular range. We don’t always search by UUID, we also search by these field sometimes to get all runs having there search Params, if that makes sense. I’m fine if you don’t think we need it but I just like to preserve data in useful formats. Saving a range as a string didn’t make sense to me and I don’t agree with your argument that since this is an exported bash variable you can’t convert it to a list using string manipulations :)

Reply:
Do we have something that works like you are describing today? I share the desire to keep data in useful formats, but I am not sure of the use case you are describing. If we keep this as a string we can still look for the strings that match for example 20-25 for node density.

Reply:
Oh, I can show you plenty of examples f sending a list/array over to ES, just maybe not in this repo. Check out the raw field in the docs at http://ospperf8.mobius.lab.eng.rdu2.redhat.com:5601/goto/0588c9b3b23122473acdbe3a9b790263 which are an array type. My whole point is
1. Just because something is exported as a bash env variable, doesn't mean we can parse it to typecaste to a list
2. I would like to see integrity of data types preserved
3. There is no such special range/array type in ES. Any field accepts a list of values as long as they are of the same type https://www.elastic.co/guide/en/elasticsearch/reference/current/array.html
4. The biggest and most important reason to have this as a list is, we need to start thinking about usecases other than where you already know the UUID and are searching results based on that. Think of the inverse, you have a bunch of data in ES and want to get the UUIDs of all runs that had a certain node range etc. While you can match on a string as well, having a list of integers gives you the flexibility to query and filter things based on range as well : https://www.elastic.co/guide/en/elasticsearch/reference/current/query-dsl-range-query.html